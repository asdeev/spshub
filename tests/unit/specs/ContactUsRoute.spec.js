import { createLocalVue, shallowMount } from '@vue/test-utils';
import { expect } from 'chai';
import Vuetify from 'vuetify';
import Vuex from 'vuex';
import initialState from '~/store/state';
import testMap from '../../testMap';

const localVue = createLocalVue();
localVue.use(Vuex);
localVue.use(Vuetify);

// TODO: Fix failing tests
describe('ContactUsRoute', () => {
  let state;

  const build = () => {
    const wrapper = shallowMount(testMap.ContactUsRoute, {
      localVue,
      state: new Vuex.Store({ state })
    });

    return {
      wrapper
    };
  };

  beforeEach(() => {
    state = { ...initialState };
  });

  it('renders the component', () => {
    const { wrapper } = build();

    expect(wrapper.html()).to.matchSnapshot();
  });
});
