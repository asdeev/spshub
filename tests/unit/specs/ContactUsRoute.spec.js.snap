// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUsRoute renders the component 1`] = `
"<div> ContactUsRoute
</div>"
`;
